.goreleaser.yml

```diff
@@ -138,21 +159,28 @@ archives:
     name_template: "tdexdconnect-v{{ .Version }}-{{ .Os }}-{{ .Arch }}"
 
 dockers:
-  # push always either release or prerelease with a docker tag with the semver only
-  - skip_push: false
+
+  ########################### 
+  # tag latest & prerelease #
+  ########################### 
+
+  #amd64
+  - image_templates:
+    - "ghcr.io/tdex-network/tdexd:{{ .Tag }}-amd64"
+     # push always either release or prerelease with a docker tag with the semver only
+    skip_push: false
+    use: buildx
     dockerfile: Dockerfile
-    # image templates
-    image_templates:
-      - "ghcr.io/tdex-network/tdexd:{{ .Tag }}"
     # GOOS of the built binaries/packages that should be used.
     goos: linux
     # GOARCH of the built binaries/packages that should be used.
     goarch: amd64
     # Template of the docker build flags.
     build_flag_templates:
+      - "--platform=linux/amd64"
       - "--pull"
       - "--label=org.opencontainers.image.created={{.Date}}"
-      - "--label=org.opencontainers.image.title={{.ProjectName}}"
+      - "--label=org.opencontainers.image.title=tdexd"
       - "--label=org.opencontainers.image.revision={{.FullCommit}}"
       - "--label=org.opencontainers.image.version={{.Version}}"
       - "--build-arg=VERSION={{.Version}}"
```

Comment:
since we defined 2 more args `TARGETOS` and `TARGETARCH` in the Dockerfile, shouldn't we add them here like for example:
```
- "--build-arg=TARGETOS=linux"
- "--build-arg=TARGETARCH=amd64"
```
?

Reply:
`buildx` injects them automatically with --platform={amd64,arm64/v8}